transmart-schemas/src/main/resources/db/changelog/changes/20190625151656_create_pg_bitcount_extension.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.1" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2019  The Hyve B.V.
+  ~  This file is distributed under the GNU General Public License
+  ~  (see accompanying file LICENSE).
+  -->
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd">
+
+    <changeSet author="ewelina@thehyve.nl" id="20190625151656-0">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="1">
+                SELECT count(*) FROM pg_available_extensions WHERE name='pg_bitcount'
+            </sqlCheck>
+        </preConditions>
+        <comment>
+            Register the pg_bitcount extension (if available) providing a bitcount function
+            and an aggregate function that aggregates integers in a bit string.
+        </comment>
+        <sql dbms="postgresql" endDelimiter=";">
+            CREATE EXTENSION pg_bitcount version '0.0.3'
```

Comment:
In transmart-data we use `CREATE EXTENSION IF NOT EXISTS pg_bitcount VERSION '0.0.3';` to make sure it does not fail if it was already installed via another way.